docs/design-language.md

```diff
@@ -7,6 +7,38 @@ import {ListControls} from "../src/components/menus";
 import {IconTabs, IndicatorColorTabs, NormalTabs, ScrollableTabs, VerticalTabs} from "../src/components/tabs";
 import {ButtonEmphasis, ButtonPairs, ButtonLabelsDo, ButtonLabelsDont, ButtonOptions, ButtonStyles, DisabledStates, ButtonColors, ButtonSizes, GroupedButtons, IconButtons, OutlinedButtons, ToggleButtons} from "../src/components/buttons";
 
+## Typograhphy
+
+Good typography plays a large role in creating not only an aesthetically pleasing app but a usable one. Therefore it is essential that style guidelines and semantics are followed.
+
+-   Don't skip heading levels. Use the headings hierarchy correctly. For example, do not follow an H1 with an H3.
+-   Overriding font styles (e.g. size and weight) is discouraged. If a custom size is required please consult with the PXD team. In most cases custom font styles are not required.
```

Comment:
I'd remove the part about consulting the PXD team.

Reply:
Updated!